wagtail_transfer/views.py

```diff
@@ -201,6 +202,7 @@ def chooser_api_proxy(request, source_name, path):
     return HttpResponse(response.content, status=response.status_code)
 
 
+@permission_required("wagtailcore.wagtailtransfer_can_import", login_url="wagtailadmin_login")
```

Comment:
I think the `chooser_api_proxy` could also do with this decorator

Reply:
Fixed in 7d97c55.